src/main/java/org/alliancegenome/curation_api/constants/EntityConstants.java

```diff
@@ -0,0 +1,8 @@
+package org.alliancegenome.curation_api.constants;
+
+public final class EntityConstants {
```

Comment:
Sorry to be weird about names but can we name this `EntityFieldNameConstants`?

Reply:
Sure, I didn't like the name either! Done.